docs/vds.rst

```diff
@@ -109,6 +110,11 @@ Reference
    Instantiate this class to represent an entire source dataset, and then
    slice it to indicate which regions should be used in the virtual dataset.
 
+   When `creating a virtual dataset <creating_vds_>`_, paths to sources present
+   in the same file are changed to a ".", refering to the current file (see
+   `set virtual <https://portal.hdfgroup.org/display/HDF5/H5P_SET_VIRTUAL>`_).
+   This will keep such sources valid is the file is renamed.
```

Comment:
This will keep such sources valid `in case` the file is renamed.

Reply:
Well spotted, thanks. I suspect 'is' was meant to be 'if'.

Reply:
> I suspect 'is' was meant to be 'if'.

Indeed :)

thanks